conference/react-js/src/components/My-talks/index.jsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
-import { connect } from 'react-redux';
+import actions from '../../constants/actions-types';
 import Talks from '../../containers/Talks';
 
 const MyTalks = () => (
-  <Talks />
+  <Talks
+    columns={[
```

Comment:
Prefer moving this array to separate constant, ahead of the component
(as is it will create a new array on every render)